spring-kafka-test/src/test/resources/junit-platform.properties

```diff
@@ -0,0 +1,3 @@
+spring.kafka.embedded.count=2
+spring.embedded.kafka.brokers.property=spring.kafka.bootstrap-servers
```

Comment:
Should this instead be `spring.kafka.embedded.broker.properties.location`?

Or is that a property not used by your `TestExecutionListener`?

Reply:
This is used by the broker itself (it tells the broker to update that system property - useful in Boot tests) - @artembilan I think we should add docs about this, especially for Boot users.

Reply:
Oh! Sorry, you just answered to Sam's question :shame: 

But yeah, I have added more in docs anyway 😄